### PR TITLE
fix function precedence order

### PR DESCRIPTION
**Pull request by ardias**

### CODE REVIEW

File: docs/basics/functions.md [hunk 1]
```diff
@@ -4,66 +4,63 @@ Flex functions can be combined in different ways to help you manipulate and tidy
...
-        -   [start_key](#startkey)
-        -   [store_lookups](#storelookups)
-        -   [strip_keys](#stripkeys)
-        -   [timestamp](#timestamp)
-        -   [to_lower](#tolower)
-        -   [value_parser](#valueparser)
-        -   [value_transformer](#valuetransformer)
+  1. [lookup_file](#lookup_file)
+  2. [start_key](#start_key)
+  3. [strip_keys](#strip_keys) \*
```
Comment (Noly) [inline docs/basics/functions.md:43]: Should the `\*` okey there?

File: docs/basics/functions.md [hunk 2]
```diff
@@ -4,66 +4,63 @@ Flex functions can be combined in different ways to help you manipulate and tidy
...
+  4. [lazy_flatten](#lazy_flatten)
+  5. [split_array (leaf_array)](#split_array)
+  6. [split_objects](#split_objects)
+  7. Standard flatten (auto-flattening)
```
Comment (Noly) [inline docs/basics/functions.md:47]: Is missing the link?

File: docs/basics/functions.md (hunk 1 above)
Comment (ardias) [reply docs/basics/functions.md:43]: yes, it's for escaping

File: docs/basics/functions.md (hunk 2 above)
Comment (ardias) [reply docs/basics/functions.md:47]: this is an internal function that is automatically applied so there is no documentation link here

**Pull request merged**